modules/nw-multinetwork-sriov.adoc

```diff
@@ -29,6 +29,26 @@ scheduled to worker nodes that have sufficient resources available.
 * The SR-IOV CNI plug-in plumbs VF interfaces allocated from the SR-IOV device
 plug-in directly into a Pod.
 
+You can use the {product-title} console to install SR-IOV, by deploying,
```

Comment:
SR-IOV Operator will be the only interface user needs to interact with, perhaps move above two paragraphs about sriov network device plugin and sriov cni after introducing all the operator features. For example, maybe have detailed introduction of each component that managed by SR-IOV operator after listing the operator features.

Reply:
Fixed.